### PR TITLE
Radio

### LINKED ISSUE #12: Add component : Checkbox & radio

### PR DESCRIPTION
**Pull request by bleucitron**
**Warning:** Please merge #32 before merging this one.

This PR adds the `Radio` component, which is heavily based on `Checkbox`.

Size management has been added (also to the `Checkbox` component).
The size management is a bit different from what was done before. It uses more the `styled` philosophy.
Please have a look at it and tell me what you think.

This fixes #12.

### CODE REVIEW

File: src/components/Form/Checkbox.jsx [hunk 1]
```diff
@@ -21,33 +22,42 @@ const StyledCheckbox = styled.input.attrs({
...
       border-color: ${props => props.theme.form.checkbox.borderColor};
       content: '';
-      width: 1.7rem;
-      height: 1.7rem;
-      transition: border 0.15s ease-in-out, color 0.15s ease-in-out;
+      width: ${props => props.theme.form.radio[props.size].size};
+      height: ${props => props.theme.form.radio[props.size].size};
+    }
+
+    &:after {
```
Comment (GuillaumeAmat) [inline src/components/Form/Checkbox.jsx]: You should "prefix" pseudo elements with `::`, not just one. It is a way to distinguish pseudo elements from pseudo classes.

**Review by GuillaumeAmat — CHANGES_REQUESTED** [review]
Great thank you! Just one change though, but it's good to me :)

File: src/components/Form/Checkbox.jsx (hunk 1 above)
Comment (bleucitron) [reply src/components/Form/Checkbox.jsx]: Done.

**Review by GuillaumeAmat — APPROVED** [review]

**Comment by GuillaumeAmat** [comment]
Thanks @iOiurson   o/

**Pull request merged**